### ISSUE TITLE
[Combobox] A11y audit

### ISSUE THREAD

**Issue by DanielleRameau** [sl-design-system/components#2516]
### Provide a general summary of the issue here

Do a11y audit for combobox, create new issues if there any findings.

### 👤 Your name

DR

**Comment by KatarzynaWM** [comment]
[Combobox/Multiple/Basic]

When navigating list elements with the keyboard arrows, the grey rectangle indicating the current position has too low a contrast ratio (1.1:1), making it very difficult to see. The contrast ratio should be at least 3:1. Same situation in the dark mode. 
This is the same issue as in here: https://github.com/sl-design-system/components/issues/2514#issuecomment-3319119564

<img width="448" height="346" alt="Image" src="https://github.com/user-attachments/assets/a5a56c58-c367-42e9-baac-0d10f5084278" />
